sidebar/panel.js

```diff
@@ -214,6 +214,51 @@ port.onMessage.addListener((response) => {
       iframe.width = 300;
       iframe.setAttribute("src", '/sidebar/panelfeedback.html');
       break;
+    case 'SCREENSHOT':
+      template = `
```

Comment:
Any way we can move this out into a separate function/file?  The main switch is getting really messy.